deploy/helm/flightctl/values.kind.yaml

```diff
@@ -9,13 +11,11 @@ flightctl:
     image:
       image:  localhost/flightctl-api
       pullPolicy: IfNotPresent
-    nodePort: 3443 # this is also mapped in /hack/kind_cluster.yaml as an extraPortMapping
-    agentAPINodePort: 7443 # this is also mapped in /hack/kind_cluster.yaml as an extraPortMapping
-    agentGrpcNodePort: 7444 # this is also mapped in /hack/kind_cluster.yaml as an extraPortMapping
+    nodePorts:
+      api: 3443 # this is also mapped in /hack/kind_cluster.yaml as an extraPortMapping
```

Comment:
The comments were already outdated.
I believe they now refer to the mappings in  `test/scripts/kind_cluster.yaml`

Reply:
updated